charts/app-config/values-production.yaml

```
@@ -478,6 +478,8 @@ govukApplications:
         # https://www.github.com/alphagov/govuk-puppet/pull/8041
         - name: GOOGLE_TAG_MANAGER_ID
           value: GTM-NZG8SF2
+        - name: GOOGLE_TAG_MANAGER_GA4_ID
+          value: GTM-P93SHJ4Z
```

Non-blocking but could we use the reference to the anchor for consistency with other apps. It also makes it more obvious that they share the ID.
```suggestion
          value: *publishing-design-system-gtm-id
```
Similar for the integration ones.

---

Sorry if my comment is confusing. The anchors are already defined:
- for integration: https://github.com/alphagov/govuk-helm-charts/blob/main/charts/app-config/values-integration.yaml#L658-L665
- for production: https://github.com/alphagov/govuk-helm-charts/blob/main/charts/app-config/values-production.yaml#L658-L661

So we should be able to reference them by using `*`. 

I found this comment useful to get my head around this YAML anchor shenanigans 😃 : https://stackoverflow.com/a/46644785

---

No worries! This confused me earlier too - those anchors have been defined further down the file and so can't be used if you try and use the alias first. There's some discussion in [this Slack thread](https://gds.slack.com/archives/C013F737737/p1712742344443979)

---

I've added an additional commit to remove the duplicate anchors - https://github.com/alphagov/govuk-helm-charts/pull/1947/commits/45c55bbcafdd60ea975098ef12f8f8eb520bb51f